spec/models/in_person_enrollment_spec.rb

```diff
@@ -491,4 +494,39 @@
       expect(failed_enrollment_without_notification.eligible_for_notification?).to eq(false)
     end
   end
+
+  describe 'enhanced_ipp?' do
+    before do
+      allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).and_return(usps_eipp_sponsor_id)
+    end
+
+    context 'when IdentityConfig.store.usps_eipp_sponsor_id equals the enrollment sponsor_id' do
```

Comment:
I think it might be nice to reword this so the implementation details are excluded from the description. What do you think about this?

```suggestion
    context 'when the enrollment sponsor ID is equal to the EIPP sponsor ID' do
```

Reply:
Good suggestion- resolved with commit 377af30457f2fed7acd3d0c10929b05e49529ff1